CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 ##### Enhancements
 
+* Generate shields.io badges for documentation coverage, unless
```

Comment:
nitpick, only a single badge is generated: "Generate shields.io badge for documentation coverage"